src/System.Net.WebSockets/src/System/Net/WebSockets/WebSocketException.cs

```diff
@@ -122,11 +122,6 @@ public WebSocketException(string message, Exception innerException)
         {
         }
 
-        public override void GetObjectData(SerializationInfo info, StreamingContext context)
```

Comment:
I didn't remove overrides due to the same breaking change uncertainty, but I haven't confirmed it either way.